public/index.html

```diff
@@ -38,6 +38,9 @@
   <meta name="msapplication-config" content="/img/win81/browserconfig.xml" />
 </head>
 <body class="transition-active transition-static">
+  <div class="annotation-title">
+    Вы зашли на старую версию интерфейса Фрифида. Она пока работает, но всё  интересное добавляется только в новой. <a href='https://beta.freefeed.net'>[Перейти к новой версии]</a>
```

Comment:
Href should point to the same page but with different host (e.g. on `freefeed.net/user/POSTUUID` the url should be `beta.freefeed.net/user/POSTUUID`) to allow seamless switch. See how redirect to `m.freefeed.net` works on ember loader page.